src/ggml.c

```diff
@@ -2607,7 +2607,7 @@ static struct ggml_tensor * ggml_new_tensor_impl(
         /*.nb           =*/ { 0, 0, 0, 0 },
         /*.op           =*/ GGML_OP_NONE,
         /*.op_params    =*/ { 0 },
-        /*.flags        =*/ 0,
+        /*.is_param     =*/ false,
```

Comment:
It looks like something went wrong while merging with the latest changes, this and all other uses of `flags` shouldn't be changed.

Reply:
hmm got it, will take a look

Reply:
okay I should have fixed all the instances where there were merge issues affecting `flags`. the code builds now after running `make -j4 gpt-2-backend gpt-j` so I think it should be in good shape now!